test/e2e/support/world.js

```diff
@@ -85,6 +92,52 @@ module.exports = (config) => {
     rimraf.sync(this.sandboxDir)
     mkdirp.sync(this.sandboxDir)
   }
+
+  async runBackgroundProcess (args, readyOutput = null) {
+    return new Promise((resolve, reject) => {
+      const handle = this.backgroundProcess.handle = spawn(this.karmaExecutable, args, { cwd: this.workDir })
+
+      let isSettled = false
+
+      // The handler should only handle spawn errors (e.g. invalid executable
```

Comment:
The reason for this is that the step to start the background process is considered success if process has started. Then it is running in a background and there is no way to mark Cucumber step as failed as it was already reported as successful.

Reply:
I think this comment would be more effective in the code.  Also try avoid 'this' and 'it'  as sometimes they make comments ambiguous.

Reply:
@johnjbarton I've added the information into the code and rephrased it a bit.